db/migrate/20231016115055_create_cached_aggregations.rb

```diff
@@ -0,0 +1,24 @@
+# frozen_string_literal: true
+
+class CreateCachedAggregations < ActiveRecord::Migration[7.0]
+  def change
```

Comment:
As discussed before, ideally we should also add `charge_id` field. In case of plan with multiple charges, where two of them belong to the same BM and where one is prorated and another one is not, we want to distinct those two cases (in case of prorated charge there is extra metadata field) and IMO we should have two different rows in `cached_aggregations` table